runtime/runtime/src/lib.rs

```diff
@@ -1568,6 +1568,84 @@ impl Runtime {
         Ok(local_receipts)
     }
 
+    /// This function wraps [Runtime::process_receipt]. It adds a tracing span around the latter
```

Comment:
nit: I think you also need the backticks to make the reference work. like ``` [`Runtime::process_receipt`] ```. Anyway as long as it works (is clickable) in vs code I'm happy :)

Reply:
I tried `cargo doc` and got lots of unrelated warnings, but this syntax worked :)